sakura_core/CDicMgr.cpp

```diff
@@ -70,6 +70,7 @@ BOOL CDicMgr::Search(
 	}
 
 	wchar_t	szLine[LINEREADBUFSIZE];
+	size_t szDelimit = wcslen(pszDelimit);
```

Comment:
You can substitute it with the following code.
```suggestion
	const auto szDelimit = _countof(pszDelimit) - 1;
```
See declaration of `pszDelimit`. at line 55.
For the variable name `szDelimit`, it is preferable to use `cchDelimit`.
The prefix `sz` means 'a string ending in NUL', which is confusing because it is usually the name given to a character array variable.

I think that the complete correction would be as follows. (arround line 55)
```c++
	constexpr auto&	pszDelimit = L" /// ";
	constexpr auto cchDelimit = _countof(pszDelimit) - 1;
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17